samcli/commands/test_runner/init/cli.py

```diff
@@ -0,0 +1,199 @@
+"""
+CLI command for "test-runner init" command
+"""
+from asyncore import write
+from email.mime import image
+import click
+import logging
+from typing import Optional, Any
+from samcli.lib.test_runner.utils import get_image_uri
+from samcli.lib.test_runner.test_runner_template_generator import FargateRunnerCFNTemplateGenerator
+from samcli.commands.exceptions import NoResourcesMatchGivenTagException
+from samcli.cli.main import pass_context
+from samcli.lib.utils.boto_utils import get_boto_client_provider_with_config
+from samcli.lib.test_runner.utils import write_file
+from samcli.lib.test_runner.utils import query_tagging_api
+from samcli.lib.test_runner.generate_env_vars import FargateRunnerArnMapGenerator
+from samcli.lib.utils.colors import Colored
```

Comment:
Should these be sorted?

Can we combine the imports from samcli.lib.test_runner.utils into one line? E.g., `from samcli.lib.test_runner.utils import get_image_uri, query_tagging_api, write_file`?

Reply:
Sure, I'll organize the imports